portfolio/src/main/webapp/script.js

```diff
@@ -92,25 +92,30 @@ function getGuestBook() {
   const guestBook = document.getElementById('guest-book-comments');
   fetch("/data").then(response => response.json()).then((comments) => {
     comments.forEach((comment) => {
-      const commentHolder = document.createElement('div');
-      const commentText = document.createElement('p');
-      const commentAuthor = document.createElement('p');
-      
-      commentHolder.className = "comment";
-      commentText.className = "message";
-      commentAuthor.className = "author";
-
-      commentText.innerText = comment.text; 
-      commentAuthor.innerText = comment.author;
-
-      commentHolder.appendChild(commentText);
-      commentHolder.appendChild(commentAuthor);
-      guestBook.appendChild(commentHolder);
-      guestBook.appendChild(document.createElement('br'));
+      guestBook.appendChild(createComment(comment));
```

Comment:
Since you are appending it might be a good idea to empty out the parent container (guestbook) to make sure existing entries are cleared out. See https://gabrieleromanato.name/javascript-empty-a-dom-element for a couple of ways to do it.

Reply:
Yes! I am working on this for the next pull request, which allows for changing how many comments are visible (which requires emptying out the guestbook). Thank you for the link!